include/batty.h

```diff
@@ -0,0 +1,8 @@
+#ifndef GUARD_BATTY_H
+#define GUARD_BATTY_H
+
+#include "data.h"
+
+struct Object2* CreateBatty(struct Object* arg0, u8 arg1);
```

Comment:
As it's placed in the function pointer array, the return type should be `void *`.